.teamcity/_Self/GoogleClassroom/Project.kt

```diff
@@ -21,6 +21,7 @@ object GoogleClassroomProject : Project({
             +:.teamcity => .teamcity
             +:%project.directory% => %project.directory%
             +:eng => eng
+			+:./src/lib => .src/lib

```

Comment:
look at those extra spaces! 

😆 

I don't really care.

Reply:
Actually, this is a good catch. They were tabs inserted by Notepad++.  Serves me right for not using VS Code!